config/zones/IN-KA.yaml

```diff
@@ -83,10 +83,8 @@ emissionFactors:
         source: Electricity Maps, 2021 average
         value: 459.2042079468457
     unknown:
-      _comment: 'Source: Souther Regional Load Dispatch Center Allocation Limits'
-      _url:
-        - http://srldc.org/2018-19srallocation.aspx
-      source: calculated 20% nuclear, 80% thermal (coal)
+      _comment: calculated 20% nuclear, 80% thermal (coal)
+      source: Souther Regional Load Dispatch Center Allocation Limits 2018-2019; SRLDC 2018-2019 derived calculation for Karnataka
```

Comment:
@zuron7 this is data you collected a few years ago now, but I wondered whether you know where there might be updated data for Karnataka from SRLDC (or elsewhere)?  It looks like their website has moved from `srldc.org` to `srldc.in`, and I'm having difficulty accessing it.